app/views/devise/passwords/new.html.erb

```diff
@@ -14,10 +14,14 @@
       url: user_password_path,
       html: { autocomplete: 'off', method: :post },
     ) do |f| %>
-  <%= f.input :email,
-              required: true,
-              input_html: { autocorrect: 'off',
-                            aria: { invalid: false, describedby: 'email-description' } } %>
+  <%= render ValidatedFieldComponent.new(
+        form: f,
+        name: :email,
+        label: t('account.index.email'),
+        required: true,
+        input_html: { autocorrect: 'off',
+                      aria: { invalid: false, describedby: 'email-description' } },
```

Comment:
`ValidatedFieldComponent` is already handling `aria-invalid` so we could remove it.

```suggestion
                      aria: { describedby: 'email-description' } },
```